templates/generic/tag_multiselect.html

```diff
@@ -0,0 +1,115 @@
+<div x-data="dropdown()" x-init="loadOptions()">
+  <input name="values" type="hidden" x-bind:value="selectedValues()">
+
+  <div id="linkTagDiv" data-url="{% url 'annotations:link_tag' object_id=object_id %}"></div>
+  <div id="unlinkTagDiv" data-url="{% url 'annotations:unlink_tag' object_id=object_id %}"></div>
```

Comment:
Are these used for anything other than storing the links? If not you could just put the links directly into the JS.

Reply:
Do you mean inject it into the JS script like this?
```JS
<script>
let url = {% url 'annotations:link_tag' %}
</script>
```

Reply:
yes, though with quotes to make it a string

Reply:
Tried that, it's not rendering properly. See https://github.com/dimagi/open-chat-studio/pull/318/commits/602e57f308287fe5f68002075d30c25e769f84c4. I extracted the JS code into a dedicated .js file so it's easy to reuse in the future. For this though, I put the urls on the script tag as data attributes, since we cannot render stuff in that JS file like we do with html templates. I'd be glad if there is a way to still do it the "template way", but I couldn't find something in my brief search

Reply:
using the data tags seems like a good compromise to me